go/coinstacks/cosmos/api/api.go

```diff
@@ -107,9 +107,15 @@ func docsRedirect(w http.ResponseWriter, r *http.Request) {
 	http.Redirect(w, r, "/docs/", http.StatusFound)
 }
 
-func (a *API) Start(errChan chan<- error) {
+func (a *API) Serve(errChan chan<- error) {
 	logger.Info("serving application")
-	go a.mananger.Start()
+
+	if err := a.handler.StartWebsocket(); err != nil {
```

Comment:
Comments would be useful to know which websocket we are interacting with (server -> cosmos, vs client -> server)